Author: Gert Wollny <gewo@debian.org>
Description: Add VTK defines for C++11 constructs to the swig interface file
--- a/Utilities/VTK/vtkgdcm.i
+++ b/Utilities/VTK/vtkgdcm.i
@@ -31,12 +31,17 @@
 #endif
 
 %{
-//#define VTK_MAJOR_VERSION 6
-//#define VTK_MINOR_VERSION 2
-//#define VTK_BUILD_VERSION 0
-//#define VTK_VERSION "6.2.0"
+//#define VTK_MAJOR_VERSION 7
+//#define VTK_MINOR_VERSION 1
+//#define VTK_BUILD_VERSION 1
+//#define VTK_VERSION "7.1.1"
 %}
 
+#define VTK_OVERRIDE override
+#define VTK_FINAL final
+#define VTK_DELETE_FUNCTION = delete
+#define VTK_NEWINSTANCE
+#define VTK_LEGACY(X)
 
 %{
 // Let's reproduce the stack of include, when one would include vtkSetGet:
@@ -118,7 +123,6 @@
 #endif
 
 
-
 #define GDCM_EXPORT
 #define VTK_EXPORT
 #define VTK_COMMON_EXPORT
